study-builder/studies/osteo/study.conf

```diff
@@ -1111,6 +1115,11 @@
           "activityCode": "ABOUTYOU",
           "expression": """user.studies["CMI-OSTEO"].forms["ABOUTYOU"].isStatus("CREATED", "IN_PROGRESS")"""
         },
+        {
```

Comment:
The workflow here is the thing I am least sure about. Welcome extra scrutiny to this part.

Reply:
Workflow changes here looks fine to me. Consider also adding `FAMILY_HISTORY` to the list for the `RETURN_USER` workflow below as well.

Did we get the official word on where family history fits into the picture?

Reply:
As I indicated above, I don't think CMI has given consideration how this feature fits the bigger picture. We'll have to press the issue.

Reply:
Got confirmation from Erin and Jen about location of family history in workflow. Added return_user entry for family history here: 72d48b8f